lib/supplejack/more_like_this_record.rb

```diff
@@ -13,8 +13,8 @@ def initialize(id, options = {})
 
       raise(Supplejack::MalformedRequest, "'#{id}' is not a valid record id") if @id <= 0
 
-      @params = { frequency: options[:frequency] || 1 }
-      @params[:mlt_fields] = options[:mlt_fields].join(',') unless options[:mlt_fields].blank?
+      @params = options&.reverse_merge(option_defaults) || option_defaults
```

Comment:
Any specs to update?

Reply:
Can `option_defaults` be a variable?

```ruby
option_defaults = { frequency: 1 }
@params = options...
```

Reply:
I actually added a spec as this class now supports extra parameters like `page` `per_page` or any given parameters in the options param.